iocore/net/EventIO.h

```diff
@@ -97,11 +74,6 @@ struct EventIO {
      @param events a mask of flags (for details `man epoll_ctl`)
      @return int the number of events created, -1 is error
    */
-  int start(EventLoop l, DNSConnection *vc, int events);
-  int start(EventLoop l, NetAccept *vc, int events);
-  int start(EventLoop l, NetEvent *ne, int events);
-  int start(EventLoop l, UnixUDPConnection *vc, int events);
-  int start(EventLoop l, int fd, NetEvent *ne, int events);
   int start_common(EventLoop l, int fd, int events);
```

Comment:
Looks like we can make `start_common` a protected member.

Reply:
done